Ejercicio 15 GIT

ijchavez opened the pull request:
Adicionalmente se usa una clase llamada seleniumUtils no presentada pero es la siguiente:

package Clase9.Ejercicio;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class seleniumUtils {
    public static WebDriver getUrlDriver(String unaURL) {
        System.setProperty("webdriver.chrome.driver", "drivers/chromedriver");
        WebDriver driver = new ChromeDriver();
        //driver = new ChromeDriver();
        driver.get(unaURL);

        return driver;
    }
}

ijchavez committed:
Se adjunta el ejercicio 15 GIT

Adicionalmente se usa una clase llamada seleniumUtils no presentada pero es la siguiente:

package Clase9.Ejercicio;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class seleniumUtils {
    public static WebDriver getUrlDriver(String unaURL) {
        System.setProperty("webdriver.chrome.driver", "drivers/chromedriver");
        WebDriver driver = new ChromeDriver();
        //driver = new ChromeDriver();
        driver.get(unaURL);

        return driver;
    }
}

ijchavez commented:
Perdon Emiliano, no habia agregado a seleniumcurso como revisor pero levante el pull request hace varios dias

seleniumcurso commented on Practico9Ejercicio15/Ejercicio15.java:
```diff
@@ -0,0 +1,105 @@
+
+        driver.manage().window().maximize();
+
+        List<WebElement> div = driver.findElements(By.tagName("div"));
```
mejorar el nombre de las listas, por ejemplo: listaDivs

ijchavez replied:
Hecho

seleniumcurso commented on Practico9Ejercicio15/Ejercicio15.java:
```diff
@@ -0,0 +1,105 @@
+        String titulo = driver.getTitle();
+        System.out.println("El titulo de la pagina es: " + titulo);
+
+        List<WebElement> input = driver.findElements(By.tagName("input"));
```
mejorar el nombre de las listas, por ejemplo: listaInputs

ijchavez replied:
hecho

seleniumcurso commented on Practico9Ejercicio15/Ejercicio15.java:
```diff
@@ -0,0 +1,105 @@
+
+            }
+        }
+        System.out.println("El elemento que mas se repite es " + max);
```
esta correcto pero como se haria para saber cual es el o los elementos que mas se repitieron?

ijchavez replied:
Hecho. Gracias por tus comentarios!

ijchavez committed:
Correcciones

- Se mejoraron los nombres de las listas.
- Se crea la identificación de elemento que mas se repitió
